runtime/common/computationkind.go

```diff
@@ -109,7 +109,7 @@ const (
 	_
 	_
 	_
-	_
+	ComputationKindEncodeValue
```

Comment:
I added this new computation kind here. Let me know if this needs to change. We could remove the computation for `ComputationKindSetValue` from FVM once this is merged.
cc: @janezpodhostnik